### PR TITLE
Add --force flag to pest:test command

### PR DESCRIPTION
**Pull request by rezaamini-ir**
From now on with this PR we will be able to create a test with `--flag` and recreate the test file.

### CODE REVIEW

File: src/Laravel/Commands/PestTestCommand.php [hunk 1]
```diff
@@ -21,7 +21,7 @@ final class PestTestCommand extends Command
      *
      * @var string
      */
-    protected $signature = 'pest:test {name : The name of the file} {--unit : Create a unit test} {--dusk : Create a Dusk test} {--test-directory=tests : The name of the tests directory}';
+    protected $signature = 'pest:test {name : The name of the file} {--unit : Create a unit test} {--dusk : Create a Dusk test} {--test-directory=tests : The name of the tests directory} {--force : Force create}';
```
Comment (lukeraymonddowning) [inline src/Laravel/Commands/PestTestCommand.php:24]: Perhaps a more descriptive description?

"Overwrite any existing test file with the same name" or something similar

**Comment by rezaamini-ir** [comment]
@lukeraymonddowning Now, What do you think about the description?

**Comment by lukeraymonddowning** [comment]
Much better @rezaamini-ir

**Comment by nunomaduro** [comment]
@rezaamini-ir Can you address the static analysis errors?

**Comment by rezaamini-ir** [comment]
@nunomaduro What do you mean by 'static analysis errors'?

**Comment by rezaamini-ir** [comment]
@nunomaduro I guess with these changes it will be fixed.

**Pull request merged**